rotors_driver/test/test_pubs.py

```diff
@@ -0,0 +1,97 @@
+#! /usr/bin/env python
+
+import os
+import rospy
+from mavros_msgs.msg import State, ExtendedState
+from geometry_msgs.msg import PoseStamped, TwistStamped
+from sensor_msgs.msg import BatteryState, NavSatFix
+
+ns_bat = 'mavros/battery'
+ns_state = 'mavros/state'
+ns_ext_state = 'mavros/extended_state'
+ns_pose = 'mavros/local_position/pose'
+ns_vel = 'mavros/local_position/velocity_body'
+ns_global = 'mavros/global_position/global'
+
+RATE = 10
+CONT = 0
+TEMP = "{}\n" \
+       "\n" \
+       "###########################\n" \
+       "#      TELLO STATUS       #\n" \
```

Comment:
Tello?

Reply:
> Please remove the unnecessary comments and spaces. The "firefly" model is missing inside drone_assets.

Sure, will do the cleaning

Reply:
> Tello?

Oh! I have to remove the unnecessary code.  Actually I took tello_driver tests as reference for a quick test of rotors_driver before adding it to the RADI